src/item.js

```diff
@@ -14,21 +25,30 @@ itemObj.getSchema = function () {
     isMembers: Boolean,
     buyLimit: Number,
 
-    currentTrend: String,
-    todaysTrend: String,
-    days30Trend: String,
-    days90Trend: String,
-    days180Trend: String,
-    currentPrice: Number,
-    todaysChange: Number,
-    days30Change: Number,
-    days90Change: Number,
-    days180Change: Number
+    today: {
+      trend: String,
+      amountChanged: Number,
+      price: Number
+    },
+    days30: {
+      trend: String,
+      amountChanged: Number
+    },
+    days90: {
+      trend: String,
+      amountChanged: Number
+    },
+    days180: {
+      trend: String,
+      amountChanged: Number
+    },
+
+    history: Array
   });
 };
 
 itemObj.parseItem = function (item) {
-  var prices = item.prices;
+  var rsItem = !!item.current;
```

Comment:
Change item name when rs updates their api
Rename rsItem because it's not immediately clear it's a boolean